recipes/arxiv-miner/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "arxiv-miner" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arxiv_miner-{{ version }}.tar.gz
+  sha256: 8239aafe164bf3791a3113ba6ddbc1d08dead0cc1bc87b86efc9e184e01895df
+
+build:
+  skip: true  # [win]
```

Comment:
This looks like a noarch package. If the reason to skip Windows is a missing dependency we can still make it noarch and "outsource" the install failure to the solver.

Reply:
I think this one included scripts that's why I didn't put noarch, and the scripts are for unix-like only